src/main/java/com/setycz/chickens/ChickensRegistryItem.java

```diff
@@ -98,7 +113,7 @@ public int getDyeMetadata() {
     }
 
     public boolean canSpawn() {
-        return getTier() == 1 && !isDye();
+        return getTier() == 1 && (!isDye() || isDye(EnumDyeColor.WHITE.getDyeDamage()));
```

Comment:
I really liked this change, I didn't think about that it's not possible to obtain bones in peaceful.